### PR TITLE
Allow use and testing with Jekyll 4.0

### PR DESCRIPTION
**Pull request by ashmaroli**
Jekyll 4.0 is on the horizon..

### CODE REVIEW

**Comment by ashmaroli** [comment]
@benbalter I didn't realize all these tasks would be within the scope of this one pull request. Sorry for the trouble..

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 language: ruby
 rvm:
-  - 2.3.0
+  - 2.5
 before_install: gem install bundler -v 1.11.2
```
Comment (ashmaroli) [inline .travis.yml]: This line is redundant now..
```suggestion
```

**Comment by benbalter** [comment]
>  I didn't realize all these tasks would be within the scope of this one pull request. Sorry for the trouble..

That's how things always seem to go! :smile: Really appreciate the PR. Thanks!

**Comment by benbalter** [comment]
@ashmaroli Would love to get this merged if you have a spare cycle to resolve the Rubocop issue, if not, no worries. Appreciate the PR!

**Comment by ashmaroli** [comment]
> if you have a spare cycle to resolve the Rubocop issue,

I'll look into it. No issues..

**Comment by ashmaroli** [comment]
*Requires #33*..

**Comment by ashmaroli** [comment]
I recommend a `squash-merge` on this one. Too many back-n-forth commits..

**Comment by benbalter** [comment]
Thanks again @ashmaroli!

**Comment by ashmaroli** [comment]
You're welcome @benbalter :smile:

**Comment by ashmaroli** [comment]
@benbalter Jekyll 4.0 is around the corner. Hope you can a cut a release to ship support for Jekyll 4.0 soon.
Thanks.

**Pull request merged**